### ISSUE TITLE
Subdocument change history support?

### ISSUE THREAD

**Issue by dandv** [nosuchip/mongoose-track#6]
Is [this](https://github.com/nassor/mongoose-history/issues/33) supported? If so, would be helpful to mention in the README. The README does say "deeply" but it wasn't very clear to me what that means.

**Comment by nosuchip** [comment]
@dandv yes, history document store changes recursively. Actually README is pretty close to original one, I almost didn't updated it. Thanks for info.
